harvard-imperial-college-london.csl

```diff
@@ -90,6 +91,7 @@
       <choose>
         <if type="report" match="none">
           <text variable="publisher"/>
+          <text variable="publisher-place"/>
```

Comment:
There is no delimiter between publisher and publisher-place here. You should put these inside a `<group delimiter=", ">`